skaffold.yaml

```diff
@@ -0,0 +1,77 @@
+# Skaffold - https://skaffold.dev/
+# ------------------------------- #
+# Installation options & profiles #
+# ------------------------------- #
+# - Istio    - `skaffold run`
+# - Gateways - `skaffold run -p ingress,egress`
+# - Addons:
+#   - kiali  - `skaffold run -p kiali`
+# ------------------------------- #
+# Development mode - skaffold dev #
+# ------------------------------- #
+apiVersion: skaffold/v2beta22
+kind: Config
+metadata:
+  name: istio-base
+deploy:
+  helm:
+    releases:
+      - name: istio-base
+        chartPath: manifests/charts/base
+        namespace: istio-system
+        createNamespace: true
+---
+apiVersion: skaffold/v2beta22
+kind: Config
+metadata:
+  name: istiod
+requires:
+  - configs: [istio-base]
+deploy:
+  helm:
+    releases:
+      - name: istiod
+        chartPath: manifests/charts/istio-control/istio-discovery
+        namespace: istio-system
+---
+apiVersion: skaffold/v2beta22
+kind: Config
+metadata:
+  name: istio-gateways
+requires:
+  - configs: [istiod]
+profiles:
+  - name: ingress
+    deploy:
+      helm:
+        releases:
+          - name: istio-ingressgateway
+            chartPath: manifests/charts/gateways/istio-ingress
+            namespace: istio-ingressgateway
+            createNamespace: true
+  - name: egress
+    deploy:
+      helm:
+        releases:
+          - name: istio-egressgateway
```

Comment:
suggest using https://artifacthub.io/packages/helm/istio-official/gateway, which are the beta supported gateway charts.